File: ts/wam/3206.ts
export const WamGraphqlCatalogRequest = 3206 as const

export interface GraphqlCatalogRequestEvent {
  /** field 1, wire `graphql_catalog_endpoint` */
  graphqlCatalogEndpoint?: GRAPHQLCATALOGENDPOINT
  /** field 2, wire `graphql_request_result` */
  graphqlRequestResult?: GRAPHQLREQUESTRESULT
  /** field 3, wire `graphql_error_code` */
  graphqlErrorCode?: number
  /** field 4, wire `business_jid` */
  businessJid?: string
  /** field 5, wire `business_type` */
  businessType?: BUSINESSTYPE
}

File: go/wam/3206.go
const WamGraphqlCatalogRequest = 3206 // channel: private

type GraphqlCatalogRequestEvent struct {
	GraphqlCatalogEndpoint GRAPHQLCATALOGENDPOINT `wam:"graphql_catalog_endpoint"` // field 1
	GraphqlRequestResult GRAPHQLREQUESTRESULT `wam:"graphql_request_result"` // field 2
	GraphqlErrorCode int64 `wam:"graphql_error_code"` // field 3
	BusinessJid string `wam:"business_jid"` // field 4
	BusinessType BUSINESSTYPE `wam:"business_type"` // field 5
}